.npmignore

```diff
@@ -0,0 +1 @@
+src
```

Comment:
I think you could add `node_modules` here

Reply:
This is automatically done